src/main/kotlin/no/nav/klage/search/api/mapper/AccessMapper.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.klage.search.api.mapper
+
+import no.nav.klage.kodeverk.Ytelse
+import no.nav.klage.search.api.view.AccessView
+import no.nav.klage.search.domain.elasticsearch.EsAnonymKlagebehandling
+import no.nav.klage.search.service.saksbehandler.InnloggetSaksbehandlerService
+import no.nav.klage.search.service.saksbehandler.OAuthTokenService
+import org.springframework.stereotype.Component
+
+@Component
+class AccessMapper(
+    private val oAuthTokenService: OAuthTokenService,
```

Comment:
Jeg har dratt avhengigene inn her, i stedet for å ha dem i Controlleren og sende resultatet inn til mapperen. Ok?